test/TestBase.py

```diff
@@ -190,4 +203,36 @@ def assertAccessDependency(self,
             self.remove(items)
 
     def assertBeatable(self, beatable: bool):
+        """Asserts that the game can be beaten with the current state"""
         self.assertEqual(self.multiworld.can_beat_game(self.multiworld.state), beatable)
+
+    # following tests are automatically run
+    @property
+    def dont_run_default_tests(self) -> bool:
```

Comment:
imo this should either be `run_default_tests` or `skip_default_tests`, not `dont_run`

Reply:
i honestly just couldn't think of a good name lmao